modules/core/src/main/java/org/locationtech/jts/geom/LinearRing.java

```diff
@@ -132,13 +132,12 @@ protected LinearRing copyInternal() {
     return new LinearRing(points.copy(), factory);
   }
 
-  /** @deprecated */
-  public Geometry reverse()
+  public LinearRing reverse()
```

Comment:
This is no longer deprecated, a change that was producing warnings in client code.